cpp/log/sqlite_db-inl.h

```diff
@@ -4,12 +4,16 @@
 
 #include "log/sqlite_db.h"
 
+#include <gflags/gflags.h>
 #include <glog/logging.h>
 #include <sqlite3.h>
 
 #include "log/sqlite_statement.h"
 #include "util/util.h"
 
+DEFINE_bool(sqlite_synchronous_off, false,
```

Comment:
Worth a TODO somewhere to capture the conversation we had just now about turning sync off while a fresh node is playing catchup (to let it catch up in less than two weeks), and then turning it on when it's in the game proper?